src/main/java/EPstorage/ProfileCommands.java

```diff
@@ -0,0 +1,227 @@
+package EPstorage;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.TreeMap;
+
+
+/**
+ * class that contains all methods that deal with Profile object
+ */
+public class ProfileCommands {
+    private File genreList;
+    private UserProfile userProfile;
+    private EditProfileJson editProfileJson;
+
+    public ProfileCommands(UserProfile userProfile) throws FileNotFoundException {
+        genreList = new File("../../../../EPdata/genreIDlist.txt");
+        this.userProfile = userProfile;
+        this.editProfileJson = new EditProfileJson();
+    }
+
+    /**
+     * change name in profile
+     */
+    public void setName(String name) throws IOException {
```

Comment:
Same issue with javadoc comments. Please follow the conventions in the OSS coding standard guide.

![image](https://user-images.githubusercontent.com/35756209/67219536-cf14ed80-f45a-11e9-959c-003f8552599d.png)